org/contributors.yml

```diff
@@ -528,11 +524,9 @@ contributors:
 - satiar
 - sboyajyan
 - schaefm-ibm
-- schubert
```

Comment:
The user "schubert" leads to 422 response in peribolos for unknown reasons and blocks at the moment the org automation.
The user exists in github, is currently not an org member but I can't even invite "schubert" manually (is greyed out).

Merging this PR should unblock the org automation.